deb/publish/gen.rb

```diff
@@ -34,14 +34,15 @@
 </p>
 
 <p>
-You will need to explicitly install a Java runtime environment, <b>because Jenkins does not work with Java 9</b>, this is the safest way to
+You will need to explicitly install a Java Runtime Environment, <b>because Jenkins does not support all Java versions</b>, this is the safest way to
```

Comment:
@MRamonLeon maybe even that. Oracle is merging JDK and JRE in its distributions, so maybe we do not want to keep JRE as a term anymore

```suggestion
You will need to explicitly install Java, <b>because Jenkins does not support all Java versions</b>, this is the safest way to
```

Reply:
It looks definitively better